src/views/design/ScriptsOverview/index.tsx

```diff
@@ -188,17 +189,21 @@ const ScriptsOverview = withStyles(styles)(
                                             sortedColumn={filterFromState.sortedColumn as ISortedColumn<{}>}
                                         />
                                     </Box>
-                                    <Box flex="0 0 auto">
-                                        <Button
-                                            variant="contained"
-                                            color="secondary"
-                                            size="small"
-                                            startIcon={<AddRounded />}
-                                            onClick={() => redirectTo({ routeKey: ROUTE_KEYS.R_SCRIPT_NEW })}
-                                        >
-                                            <Translate msg="scripts.overview.header.add_button" />
-                                        </Button>
-                                    </Box>
+                                    {checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.